gprofiler/metadata/application_identifiers_java.py

```diff
@@ -40,7 +43,7 @@ def get_app_id(self, process: Process) -> Optional[str]:
             if java_command:
                 return f"java: {java_command}{' (' + ' '.join(java_args) + ')' if java_args else ''}"
         except CalledProcessError as e:
-            _logger.warning(f"Couldn't get Java properties for process {process.pid}: {e.stderr}")
+            _logger.warning("Couldn't get Java properties for process %d: %s", process.pid, e.stderr)
```

Comment:
Can fix it to newstyle extra formatting, i.e `pid=process.pid, jattach_stderr=e.stderr`